tests/lora/test_lora_layers_wan.py

```diff
@@ -137,3 +146,45 @@ def test_simple_inference_with_text_lora_fused(self):
     @unittest.skip("Text encoder LoRA is not supported in Wan.")
     def test_simple_inference_with_text_lora_save_load(self):
         pass
+
+    @require_peft_version_greater("0.13.2")
+    def test_lora_exclude_modules_for_wan(self):
+        """
+        We test if the modules from `target_modules` and `exclude_modules`, that have
+        overlap in their names, are impacted as expected. Refer to
+        https://github.com/huggingface/diffusers/pull/11806 for more details.
+        """
+        scheduler_cls = self.scheduler_classes[0]
+        components, text_lora_config, denoiser_lora_config = self.get_dummy_components(scheduler_cls)
+        pipe = self.pipeline_class(**components).to(torch_device)
+        _, _, inputs = self.get_dummy_inputs(with_generator=False)
+
+        # Only denoiser for now.
+        denoiser_lora_config.target_modules = ["to_q", "to_k", "to_v", "out"]
+        denoiser_lora_config.exclude_modules = ["proj_out"]
```

Comment:
I wonder: Would this test not pass even without the fix of this PR? When I tested it on main, it passed, even when I commented out the `denoiser_lora_config.exclude_modules = ["proj_out"]` line.

We would need an example with something like:

```python
target_modules = ["proj_out"]
exclude_modules = ["foo.proj_out"]
```

Reply:
SGTM